sql/moz-fx-data-shared-prod/telemetry_derived/events_v1/schema.yaml

```diff
@@ -0,0 +1,155 @@
+fields:
+- mode: REQUIRED
+  name: submission_date
+  type: DATE
+  description: Submission Date
```

Comment:
These descriptions seem a bit superfluous but it's fine.  Did you use bqetl to create this?

Reply:
Yes, I just added the descriptions (lazily as you can tell).